programs/ziti-edge-tunnel/CMakeLists.txt

```diff
@@ -17,6 +17,12 @@ if(CMAKE_SYSTEM_NAME STREQUAL Darwin OR CMAKE_SYSTEM_NAME STREQUAL Linux)
             PRIVATE ZITI_LOG_PREFIX=${PROJECT_NAME}
             )
 
+    # override LWIP defaults here
+    target_compile_definitions(lwipcore
+            PUBLIC MEMP_NUM_TCP_PCB=256
```

Comment:
we should add CMAKE options for these
`$ cmake -DMAX_TCP_CONNENCTIONS=1024 ...`